nvi-commons/src/main/java/no/sikt/nva/nvi/common/db/CandidateDao.java

```diff
@@ -38,23 +39,28 @@
 public final class CandidateDao extends Dao {
 
     public static final String TYPE = "CANDIDATE";
+    public static final String PERIOD_FIELD = "period";
     @JsonProperty(IDENTIFIER_FIELD)
     private final UUID identifier;
     @JsonProperty(DATA_FIELD)
     private final DbCandidate candidate;
     @JsonProperty(VERSION_FIELD)
     private final String version;
+    @JsonProperty(PERIOD_FIELD)
+    private final String period;
```

Comment:
Is the data type a String because it may be changes to some identifier instead of year in the future?